db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_24_170344) do
```

Comment:
I realized that turn time and session time were values greater than 4 bytes, so I changed their datatypes to bigints.